website/conferences/model.py

```diff
@@ -25,6 +25,19 @@ class Conference(StoredObject):
     public_projects = fields.BooleanField(required=False, default=True)
     poster = fields.BooleanField(default=True)
     talk = fields.BooleanField(default=True)
+    field_names = fields.DictionaryField(
+        default=lambda: {
+            'poster': 'poster',
+            'talk': 'talk',
+            'posters': 'posters',
+            'talks': 'talks',
+            'Posters & Talks': 'Posters & Talks',
```

Comment:
If I understand the requirement correctly, the purpose of the "poster or talk" field is to allow assets to be called something other than "poster" or "talk". 

Since these fields do seem necessary for the templates to work correctly, I'll sit down with Henrique to figure out what we can name these fields to ensure that the schema meets the needs of many conference types.